### PR TITLE
.circleci/config.yml: check mixins

### PR DESCRIPTION
**Pull request by simonpasquier**

### CODE REVIEW

**Comment by roidelapluie** [comment]
cc @beorn7 @SuperQ

File: .circleci/config.yml [hunk 1]
```diff
@@ -67,13 +68,34 @@ jobs:
         environment:
           GOGC: "20"
           GOOPTS: "-p 2 -mod=vendor"
+
+  test_mixins:
+    docker:
+    - image: quay.io/coreos/jsonnet-ci:release-0.37
```
Comment (paulfantom) [inline .circleci/config.yml:74]: This image will be deprecated soon as it doesn't have a purpose anymore. It was needed before to easily ship C++ version of `jsonnetfmt` but this binary is included in go version of jsonnet v0.16.

I recommend using separate `go.mod` file and installing tooling dependencies from there or adding `jsonnet` to circle orb.

Comment (paulfantom) [reply .circleci/config.yml:74]: Correction. This image is already deprecated.

**Comment by beorn7** [comment]
Great idea.

As @paulfantom said, I guess we should just `go install` the required tools `github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb`, `github.com/google/go-jsonnet/cmd/jsonnetfmt`, `github.com/google/go-jsonnet/cmd/jsonnet`. Do you think you can make that work?

File: .circleci/config.yml (hunk 1 above)
Comment (simonpasquier) [reply .circleci/config.yml]: :+1:

**Comment by beorn7** [comment]
The windows test failure is related to the new Go1.15 release. Apparently, the windows test uses the newest Go version automatically, and Go1.15 is stricter on X.509 certs. We need to fix that, but it's not related to this PR.

**Comment by roidelapluie** [comment]
> The windows test failure is related to the new Go1.15 release. Apparently, the windows test uses the newest Go version automatically, and Go1.15 is stricter on X.509 certs. We need to fix that, but it's not related to this PR.

Yes that's the price to pay because the default circleci uses go 1.12.

**Pull request merged**